gui/wxpython/gui_core/widgets.py

```diff
@@ -979,6 +980,55 @@ def Select(self):
         self.Refresh()
 
 
+class LayersListValidator(GenericValidator):
+    """This validator check output map existence"""
+
+    def __init__(self, condition, callback):
+        """Standard constructor.
+
+        :param condition: function which accepts string value and returns T/F
+        :param callback: function which is called when condition is not fulfilled
+        """
+        GenericValidator.__init__(self, condition, callback)
+
+    def Clone(self):
+        """Standard cloner.
+
+        Note that every validator must implement the Clone() method.
+        """
+        return LayersListValidator(self._condition, self._callback)
+
+    def Validate(self, win, validate_all=False):
+        """Validate output map existence"""
+        mapset = grass.gisenv()['MAPSET']
+        maps = grass.list_grouped(type=self._condition)[mapset]
+
+        # Check all selected layers
+        if validate_all:
+            outputs = []
+            data = win.GetLayers()
+
+            if data is None:
+                return
```

Comment:
Probably it should be `return False` here?

Reply:
I added it.